src/pages/FundManage/FundManage.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from 'react';
+import { Outlet } from 'react-router-dom';
+import FundBanner from '../../components/Fund/FundBanner';
+import openSeaLogo from "../../img/fundLogos/openSeaLogo.png";
+import netFineLogo from "../../img/fundLogos/netFineLogo.png";
+import funConLogo from "../../img/fundLogos/funConLogo.png";
+import btcImg from "../../img/icons/BTCIcon.svg";
+import ethImg from "../../img/icons/ETHIcon.svg";
+import curveImg from "../../img/icons/curveIcon.svg";
+import dopexImg from "../../img/icons/dopexIcon.svg";
+
+interface fundType {
```

Comment:
use capital letter for types. I have all these in models.ts. Move it there. You can just add the new variables into the Fund object. 

So the interface will be called Fund.

Reply:
Done